src/packages/@ncigdc/components/SurvivalPlotWrapper.js

```diff
@@ -149,11 +130,9 @@ const SurvivalPlotWrapper = ({
                 }))}
                 key="download"
                 noText
-                slug={plotName}
+                slug="survival-plot"
```

Comment:
```suggestion
                slug={plotName}
```

this will render as `slug="survival-plot"` if no custom slug is provided.